client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -155,7 +155,13 @@ export function* executeChallengeSaga({ payload }) {
     }
 
     if (challengeComplete && payload?.showCompletionModal) {
-      yield put(openModal('completion'));
+      let canOpenModal = sessionStorage.getItem('canOpenModal');
+
+      if (canOpenModal === 'false') {
+        // do nothing
+      } else {
+        yield put(openModal('completion'));
+      }
```

Comment:
```suggestion
      if (canOpenModal !== 'false') {
        yield put(openModal('completion'));
      }
```

We definitely shouldn't be using an `if/else` with an empty `if` block.

Reply:
This is removed